test/tests_basic.jl

```diff
@@ -277,10 +278,225 @@ end
     @test_nowarn p4est_destroy(p4est)
     @test_nowarn p4est_connectivity_destroy(connectivity)
   end
+end
+
+
+# This belongs to the testset "p8est_qcoord_to_vertex" below.
+function iter_volume_for_p8est_qcoord_to_vertex(info::Ptr{p8est_iter_volume_info_t}, user_data)
+  info = unsafe_load(info)
+  p4est = unsafe_load(info.p4est)
+  quad = unsafe_load(info.quad)
+
+  vxyz = Array{Float64}(undef, 3)
+  p8est_qcoord_to_vertex(p4est.connectivity, info.treeid, quad.x, quad.y, quad.z, vxyz)
```

Comment:
To follow our own guidelines, this should be

```suggestion
  info_obj = unsafe_load(info)
  p4est = unsafe_load(info_obj.p4est)
  quad = unsafe_load(info_obj.quad)

  vxyz = Array{Float64}(undef, 3)
  p8est_qcoord_to_vertex(p4est.connectivity, info_obj.treeid, quad.x, quad.y, quad.z, vxyz)
```
Could you please also adapt also the 2D version?

Reply:
Yes, I'll do that.